### ISSUE TITLE
[FR] Let user pick from a list of predefined colors for selection and cursor colors.

### ISSUE THREAD

**Issue by MayurSMahajan** [AppFlowy-IO/AppFlowy#5041]
### Description

Right now in the Settings, we let the user pick a hex color for the selection and cursor color options.
Non-technical users might not be familiar or comfortable with using direct hex colors.
So we should append a list of some 6-7 common colors as we have in every other color picker menu.

The Hex color option should not be removed, it offers amazing customizability.

### Impact

Users who are not familiar with Hex code coloring will also have an option

### Additional Context

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/47064215/6a1fb97a-99cb-40aa-9341-f6fe7abbb359)

**Comment by emmggi** [comment]
We can implement color pickers from packages:

https://pub.dev/packages/flutter_colorpicker
https://pub.dev/packages/flex_color_picker

**Comment by omprakashd2023** [comment]
Can i resolve this issue?

**Comment by MayurSMahajan** [comment]
Thanks @omprakashd2023! Assigned to you.

**Comment by omprakashd2023** [comment]
Should i fork the repo or create a new branch and work on the issue??

**Comment by ekakshjanweja** [comment]
Hey @omprakashd2023 are you still working on this?

**Comment by omprakashd2023** [comment]
@ekakshjanweja Yess, will finish within this week

**Comment by Xazin** [comment]
> @ekakshjanweja Yess, will finish within this week

Hey @OmprakashD20, I've unassigned you for now. Feel free to open a PR if you finish the improvment.

**Comment by gustamor** [comment]
Hi!

I'm working on this feature request (FR) and wanted to check if I can be assigned to it. I'll have a pull request ready soon.

**Comment by annieappflowy** [comment]
Assigned this issue to @gustamor . Thanks for helping

**Comment by gustamor** [comment]
@annieappflowy, may be I need a bit of mentoring.

I have added new entries to `assets/translations/en.json` and generated the locale files using the command `dart run easy_localization:generate -source-dir -S assets/translations -s en.json -f keys -o locale_keys.g.dart` to produce the `locale_keys.g.dart` file. However, when I run `sh ./scripts/code_generation/language_files/generate_language_files.sh` on Linux, it doesn't update `locale_keys.g.dart` with the new entries from the en.json file.

Could you please provide guidance on this issue? I understand that I only need to commit the modified files in assets/translations, correct?

So far, I have created a dialog with predefined color selections, a color wheel, shades, and accent colors, which is working fine, while maintaining the existing dialog box as requested in the FR. However, I'm unsure about handling the text strings and believe it's better to ask for guidance rather than hardcoding them.

Thank you for your help!

**Comment by gustamor** [comment]
I realized that my localization process wasn't working as expected because we use inlang. I overlooked this detail because I read the documentation too quickly.

I am now reviewing the inlang documentation and will make the necessary adjustments. I have a few questions to ensure I proceed correctly:

- Are there any additional steps or specific configurations that I should be aware of?
- What is the correct order for making the pull request with these changes?

I appreciate any guidance on using inlang effectively in AppFlowy.

Thank you for your support. I am eager to contribute correctly and learn from this experience.
